docs/docs/7-migration-guide/migration-from-v7-to-v8.md

```diff
@@ -734,17 +847,23 @@ In `package.json` update the version of the MUI X packages to `^7.22.3`.
 + "@mui/x-data-grid-premium": "^7.22.3",
 ```
 
-:::note Codemod
+A lots of props have been renamed from MUI, for a detailed look, see the official [migration guide v5 -> v6](https://mui.com/x/migration/migration-data-grid-v5) and [migration guide v6 -> v7](https://mui.com/x/migration/migration-data-grid-v6/). There is also a codemod from MUI which handles most of the changes:
 
-```sh
-npx @comet/upgrade v8/mui-x-upgrade.ts
-```
+:::note Handled by following upgrade script
+
+    ```sh
+    npx @comet/upgrade v8/mui-x-codemods.ts
+    ```
 
 :::
 
-A lots of props have been renamed from MUI, for a detailed look, see the official [migration guide v5 -> v6](https://mui.com/x/migration/migration-data-grid-v5) and [migration guide v6 -> v7](https://mui.com/x/migration/migration-data-grid-v6/). There is also a codemod from MUI which handles most of the changes:
+</details>
+
+:::warning
 
-! As well, be aware if you have a date in the data grid, you will need to add a `valueGetter`
+Be aware if you have a date in the data grid, you will need to add a `valueGetter`
```

Comment:
We have custom column types for this, should we promote them instead?

https://github.com/vivid-planet/comet/blob/next/packages/admin/admin/src/dataGrid/gridColumnTypes.tsx